settings/src/main/kotlin/voice/settings/SettingsViewModel.kt

```diff
@@ -119,4 +132,18 @@ class SettingsViewModel
     dismissDialog()
     navigator.goTo(Destination.Website("https://hosted.weblate.org/engage/voice/"))
   }
+
+  override fun toggleAutoSleepTimer(checked: Boolean) {
```

Comment:
I would not name it toggle. Toggle implies that it's being flipped but through the checked boolean it's already specifying the state